### PR TITLE
Updates for flex & grid layout

### PR DESCRIPTION
**Pull request by colepeters**
Covers #23 

- Move box alignment classes out of `flex.mjs` and into `box-align.mjs`
  - Add new classes for missing values
  - Update all classes w/ verbose naming (`align-*`, `justify-*`, etc)
  - Add classes for `place-*`
- Move order classes out of `flex.mjs` and into `order.mjs`
- Move gap classes out of `grid.mjs` and into `gap.mjs`
- Rename `align.mjs` to `text-align.mjs`

### CODE REVIEW

**Review by ryanbethel — COMMENTED** [review]
For the part of this that relates to the issue I submitted this looks great. Thanks for fixing it. For the rest which I only partially follow, LGTM 😄.

File: order.mjs
Comment (kristoferjoseph) [inline order.mjs:1]: Should order have a heading?

Comment (kristoferjoseph) [reply order.mjs:1]: I am completely unsure myself.

File: box-align.mjs
Comment (kristoferjoseph) [inline box-align.mjs:1]: These class names are pretty verbose.
Should we offer aliases that aren't?
Similar to the names for margin and padding?

File: order.mjs
Comment (colepeters) [reply order.mjs:1]: It definitely should, and I totally missed it. Thanks!

File: text-align.mjs
Comment (kristoferjoseph) [inline text-align.mjs:1]: I messed up the heading on this one 😬
Needs to be only three asterisks for the regex fairies.

Comment (colepeters) [reply text-align.mjs:1]: Fixed!

File: box-align.mjs
Comment (kristoferjoseph) [reply box-align.mjs:1]: Yes. Lets leave it verbose until someone has a better suggestion.

**Review by kristoferjoseph — APPROVED** [review]

**Pull request merged**